examples/Nat/NAT.cs

```diff
@@ -26,7 +26,7 @@
 expired. As long as it has an entry, it maps TCP segments from non-zero to
 zero ports, masquerading the client.
 
-FIXME currently we don't remove entries
+FIXME currently we don't remove entries because of RST
```

Comment:
Is this comment still current, or perhaps it should be elaborated into a couple of sentences describing the conditions for record expiry?